assets/js/googlesitekit/datastore/site/conversion-tracking.js

```diff
@@ -0,0 +1,218 @@
+	baseName: 'getConversionTrackingSettings',
+	controlCallback: () => {
+		return API.get( 'core', 'site', 'conversion-tracking', null, {
+			useCache: false,
+		} );
+	},
+	reducerCallback: settingsReducerCallback,
+} );
+
+const fetchSaveConversionTrackingSettingsStore = createFetchStore( {
+	baseName: 'saveConversionTrackingSettings',
+	controlCallback: ( { settings } ) => {
+		return API.set( 'core', 'site', 'conversion-tracking', { settings } );
+	},
+	reducerCallback: settingsReducerCallback,
+	argsToParams: ( settings ) => {
+		return { settings };
+	},
+	validateParams: ( { settings } ) => {
+		invariant(
+			isPlainObject( settings ),
+			'settings must be a plain object.'
+		);
+	},
+} );
+
+const baseInitialState = {
+	conversionTracking: {
+		settings: undefined,
+		savedSettings: undefined,
+	},
+};
+
+const baseActions = {
+	/**
+	 * Saves the Conversion Tracking settings.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {Object} Object with `response` and `error`.
+	 */
+	*saveConversionTrackingSettings() {
+		const { select } = yield getRegistry();
+		const settings = select( CORE_SITE ).getConversionTrackingSettings();
+
+		return yield fetchSaveConversionTrackingSettingsStore.actions.fetchSaveConversionTrackingSettings(
+			settings
+		);
+	},
+
+	/**
+	 * Sets the Conversion Tracking enabled status.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string} enabled Consent Mode enabled status.
+	 * @return {Object} Redux-style action.
+	 */
+	setConversionTrackingEnabled( enabled ) {
+		return {
+			type: SET_CONVERSION_TRACKING_ENABLED,
+			payload: { enabled },
+		};
+	},
+};
+
+const baseControls = {};
+
+const baseReducer = createReducer( ( state, { type, payload } ) => {
+	switch ( type ) {
+		case SET_CONVERSION_TRACKING_ENABLED:
+			state.conversionTracking.settings =
+				state.conversionTracking.settings || {};
+			state.conversionTracking.settings.enabled = !! payload.enabled;
+			break;
+
+		default:
+			break;
+	}
+} );
+
+const baseSelectors = {
+	/**
+	 * Gets the Conversion Tracking settings.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {Object|undefined} Conversion Tracking settings, or `undefined` if not loaded.
+	 */
+	getConversionTrackingSettings: ( state ) => {
+		return state.conversionTracking.settings;
+	},
+
+	/**
+	 * Gets the Consent Mode enabled status.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {boolean|undefined} Consent Mode enabled status, or `undefined` if not loaded.
+	 */
+	isConversionTrackingEnabled: createRegistrySelector( ( select ) => () => {
+		const { enabled } =
+			select( CORE_SITE ).getConversionTrackingSettings() || {};
+
+		return enabled;
+	} ),
+
+	/**
+	 * Indicates whether saving the settings is currently in progress.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {boolean} True if the settings are being saved, false otherwise.
+	 */
+	isDoingSaveConversionTrackingSettings( state ) {
```

Comment:
This one isn't needed as far as I see. Let's remove this selector.

Reply:
We are going to most likely add issue for including the Conversion Tracking into the setup Ads view, but yeah I guess we can just include this selector together with other validate submit/can submit updates in that issue once added. Removed for now